src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int AMOUNT_OF_ARRAYS = 2;
+    private static final int AMOUNT_OF_ITEMS = 10;
+    private static final int INDEX_OF_KEYS = 0;
+    private static final int INDEX_OF_VALUES = 1;
+    private Object[][] dictionary = new Object[AMOUNT_OF_ARRAYS][AMOUNT_OF_ITEMS];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        int index = checkKey(key);
+        if (index != -1) {
+            dictionary[INDEX_OF_VALUES][index] = value;
+            return;
+        }
+        if (size < AMOUNT_OF_ITEMS) {
+            dictionary[INDEX_OF_KEYS][size] = key;
+            dictionary[INDEX_OF_VALUES][size++] = value;
+        }
     }
 
     @Override
     public V get(K key) {
+        int index = checkKey(key);
+        if (index != -1) {
+            return (V) dictionary[INDEX_OF_VALUES][index];
+        }
         return null;
     }
 
     @Override
     public int size() {
+        return size;
+    }
+
+    private int checkKey(K key) {
```

Comment:
i think it's more informative name of method
```suggestion
    private int getKeyIndex(K key) {
```